Table.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export default function Table(props) {
+
+  return (
+    <table className="table"> 
+
+        <tr> 
+
+          <th>{props.children}</th> 
+
+          <th>{props.children}</th> 
+
+          <th>{props.children}</th> 
+
+        </tr> 
+
+        <tr> 
+
+          <td>{props.children}</td> 
+
+          <td>{props.children}</td> 
+
+          <td>{props.children}</td> 
+
+        </tr> 
+
+        <tr> 
+
+          <td>{props.children}</td> 
+
+          <td>{props.children}</td> 
+
+          <td>{props.children}</td> 
+
+        </tr> 
+
+        <tr> 
+
+          <td>{props.children}</td> 
+
+          <td>{props.children}</td> 
+
+          <td>{props.children}</td> 
+
+        </tr> 
+
```

Comment:
```suggestion
      {props.children}
```

@Niravprajapati1 there should be only one reference to include the children. Do you want to use the suggestion so we can wrap up this pull request?

Reply:
Yes sure 😊